### PR TITLE
Include clang-resource-headers in the install targets for the LTO build

### PR DESCRIPTION
**Pull request by dschuff**
These are all the headers in lib/clang/<version> and are necessary.
Also remove unneeded symlinks/copies of lld in the install package.

### CODE REVIEW

File: src/build.py
Comment (sbc100) [inline src/build.py:921]: Why do we have this list for lto but not for the normal build?

**Review by sbc100 — APPROVED** [review]

File: src/build.py
Comment (dschuff) [reply src/build.py:921]: Because the LTO build uses the `distribution` ninja build target instead of the default `all` target (see the next line, 922). We could (and maybe should) do this for the non-LTO build as well, but I figured I'd wait until the LTO build was stable before trying to merge them.

**Pull request merged**
